src/main/resources/config/liquibase/changelog/20240625000000_changelog.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="20240625000000" author="yassinesws">
+        <addColumn tableName="lecture_unit">
+            <column name="pyris_ingestion_state" type="VARCHAR(64)" defaultValue="NOT_STARTED">
```

Comment:
Please use an actual enum in the database! Those are supported in MySQL and Postgres and are much better in terms of performance and space